tests/extras/test_aggregate.py

```diff
@@ -492,9 +492,9 @@ def aggregate_nd(nd):
     nrows = 1000
     div = 50
     column = [i % div for i in range(nrows)]
-    matrix = [column for i in range(nd)]
+    matrix = [column] * nd
     out_types = [ltype.int] * nd + [ltype.int]
-    out_value = [[i for i in range(div)]] * nd + \
+    out_value = [list(range(div))] * nd + \
                 [[nrows // div for i in range(div)]]
```

Comment:
could be `[nrows // div] * div` too

Reply:
yeah, this one I already fixed, see next commit